ddtrace/sampler.py

```diff
@@ -53,9 +53,8 @@ class RateSampler(BaseSampler):
     """
 
     def __init__(self, sample_rate=1):
-        if sample_rate <= 0:
-            log.error("sample_rate is negative or null, disable the Sampler")
-            sample_rate = 1
+        if sample_rate < 0:
+            raise ValueError("sample_rate is negative")
```

Comment:
Let's add the actual rate here so that it'll show up in the log message when it occurs (will help with debugging).